guides/Web_Frameworks/Flask.md

```diff
@@ -0,0 +1,24 @@
+# Flask
+flask is modern python web framework which is light weight and ofcourse it is written in python.Since it does not require any tools or libraries (external) therefore it is also termed as microframework. It does not have its own components therefore it is light weight. By component we can understand in terms of data abstraction layer,form validation etc which you can see in Django if you are a hard code django lover till now .The core functionality is available from pre-existing libraries but if we want to extend the functionality we can easily do so with the help of extensions.There are many extensions which are available to add them to core flask ,few of them including form validaton,Object relational mappers etc.
```

Comment:
of course written in Python?

Reply:
@sourabh1031 : Awesome efforts! But, can you re-frame first sentence and remove ``` and ofcourse it is written in python``` as mentioned by Dhruv above!